QUEST_JOBSTEP.tsv

```diff
@@ -753,42 +753,42 @@ QUEST_JOBSTEP_20150317_000752	The disciples came to Lydia Schaffen. Afterwards,
-QUEST_JOBSTEP_20150317_000781	
-QUEST_JOBSTEP_20150317_000782	
-QUEST_JOBSTEP_20150317_000783	
-QUEST_JOBSTEP_20150317_000784	
-QUEST_JOBSTEP_20150317_000785	
-QUEST_JOBSTEP_20150317_000786	
-QUEST_JOBSTEP_20150317_000787	
-QUEST_JOBSTEP_20150317_000788	
-QUEST_JOBSTEP_20150317_000789	
-QUEST_JOBSTEP_20150317_000790	
-QUEST_JOBSTEP_20150317_000791	
+$QUEST_JOBSTEP_20150317_000756	Lydia Schaffen and Rumpelstiltskin 
+$QUEST_JOBSTEP_20150317_000757	{s35}Lydia Schaffen and Rumpelstiltskin {/} 
+$QUEST_JOBSTEP_20150317_000758	This is a story about Lydia Schaffen have been handed down throughout the Kingdom. Furthermore, while there are places that Lydia Schaffen have never been to according to historical records, this story stems from a time when she visited their village. It is the kind of story that contains absurd things comparable to dragons appearing, such as the manifestation of strange demons that we seemingly recognize, and discusses the challenges Lydia Schaffen took as well as didn't take.
+$QUEST_JOBSTEP_20150317_000759	In spite of that, the villagers from the region believe the story beyond doubt, and they embrace the pride that this story about Lydia Schaffen brings, having originated from their hometown.
+$QUEST_JOBSTEP_20150317_000760	Compared to Ruklys, who likely died early on during the Civil War, or Flurry, who rarely leaves the tower she built, Lydia Schaffen definitely went on a greater number of adventures around the world, and the stories from her adventures include characters that she was well acquainted with. From them, even an adversary's fable could be a part of her stories. 
+$QUEST_JOBSTEP_20150317_000761	It is not known how true the story really is. Perhaps it was just as absurd in the past as it would definitely be absurd in the present. Still, this kind of story brings a perspective in life, and the fact that it is still told today means the story agrees with the nature of Lydia Schaffen.
+$QUEST_JOBSTEP_20150317_000762	One day Lydia Schaffen sneaked out of a certain village alone and walked around at night.{nl} Lydia Schaffen saw a traveler sitting on a woodland road. {nl} Out of curiosity, she decided to ask him.{nl} 'Why are you sitting here in the forest in the middle of the night?'{nl}The traveler was hesitant to answer up to this point, but after being asked multiple times he finally opened his mouth. {nl}The reason was apparently this. {nl}
+$QUEST_JOBSTEP_20150317_000763	The traveler had met a passing demon, named Rumpelstiltskin, in the forest several days ago. He was invited to play a game with the demon. 
+$QUEST_JOBSTEP_20150317_000764	The reward for winning the game was significant, but the game with the demon was not easy. In the end, the traveler lost to Rumpelstiltskin. And when traveler reached the part where he had to pay for losing the game, he took a short breath, he was told to keep waiting. After hearing the conditions, Lydia Schaffen started to search for the troublemaker, Rumpelstiltskin.
```

Comment:
he took a short breath, he was told to keep waiting -> he took a short breath, and was told to keep waiting